seismicpro/src/gather/gather.py

```diff
@@ -1120,6 +1120,96 @@ def resample(self, new_sample_rate, kind=3, anti_aliasing=True):
         self.samples = new_samples
         return self
 
+    @batch_method(target="for")
+    def apply_agc(self, window_size=250, mode='rms'):
+        """Calculate instantaneous of RMS amplitude AGC coefficients and apply them to gather data.
+
+        Parameters
+        ----------
+        window_size : int
```

Comment:
```suggestion
        window_size : int, optional, defaults to 250
```
Similar in other cases